client/src/components/Header/Header.css

```
@@ -1,6 +1,155 @@
-header > nav > ul {
-  display: flex;
-  list-style: none;
-  gap: 10px;
-  padding-left: 0; /* reset */
-}
\ No newline at end of file
+/* * {
+	border: 1px dotted black;
+} */
+
+header {
+	background-color: var(--gray80);
+}
+
+header nav {
+	position: fixed;
+	top: 0;
+	left: 0;
+	height: 100vh;
+	width: 100vw;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	background-color: #adadade9;
```

So we can easier fine tweak the transparency level with the alpha value (currently 0.92 (1 being opaque))

```suggestion
	background-color: hsla(0, 0%, 70%, 0.92);
```